models/media_stats.sql

```diff
@@ -1,126 +1,131 @@
-{{ 
-  config(
-    materialized='table',
-  )
-}}
-
 with prep as (
 
- 	select 
+  select
     p.media_id,
     p.duration,
-    p.title,
     p.media_type,
     p.media_player_type,
     sum(p.play_time_sec) / 3600 as play_time_h,
     avg(p.play_time_sec / 60) as avg_play_time_min,
+    sum(case when is_played then 1 else 0 end) as plays,
     sum(case when p.is_complete_play then 1 else 0 end) as complete_plays,
-    count(distinct p.page_view_id) as views,
-    count(distinct p.domain_sessionid) as "sessions",
+    count(distinct p.page_view_id) as impressions,
     avg(coalesce(p.play_time_sec / nullif(p.duration, 0), 0)) as retention_rate,
-    count(distinct p.domain_userid) as unique_users
+    coalesce(sum(p.avg_playback_rate * p.play_time_sec) / nullif(sum(p.play_time_sec), 0), 0) as avg_playback_rate
 
   from {{ ref("plays_by_pageview") }} p
- 
-  group by 1, 2, 3, 4, 5
+
+  group by 1, 2, 3, 4
 
 )
 
 , valid_play_stats as (
 
-    select
-      media_id,
-      min(start_tstamp) as first_play,
-      max(start_tstamp) as last_play,
-      count(*) as valid_plays
-    
-    from {{ ref("plays_by_pageview") }}
+  select
+    media_id,
+    min(start_tstamp) as first_play,
```

Comment:
I think both could be useful, especially now that we decided to reconsider the playtime calculation so that valid plays will become less accurate in certain scenarios